WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:47:36
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:47:36] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:47:41
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:47:41] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:48:07
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:48:07] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:48:36
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:48:36] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.157:5000 (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:48:51
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:48:51] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:49:49
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:49:49] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:49:58
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:49:58] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:50:13
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:50:13] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                            FROM rt_winback_config
                            ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
ERROR:routes.rt_winback:Replacement index 0 out of range for positional args tuple
ERROR:routes.rt_winback:Exception thrown:Replacement index 0 out of range for positional args tuple
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:50:19
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:50:19] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.157:5000 (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= 2021-01-01 
                            AND date_installed <= 2022-04-09
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:51:44
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:51:44] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:53:15
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:53:15] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.157:5000 (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM rt_winback_existing_df rt, orders_order oo
                            WHERE oo.id == rt.order_id
                            AND date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:ROLLBACK
ERROR:routes.rt_winback:(pymysql.err.ProgrammingError) (1064, 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'== rt.order_id\n                            AND date_installed >= "2020-01-01" \n \' at line 3')
[SQL: SELECT id, btn, campaign
                            FROM rt_winback_existing_df rt, orders_order oo
                            WHERE oo.id == rt.order_id
                            AND date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"]
(Background on this error at: https://sqlalche.me/e/14/f405)
ERROR:routes.rt_winback:Exception thrown:(pymysql.err.ProgrammingError) (1064, 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'== rt.order_id\n                            AND date_installed >= "2020-01-01" \n \' at line 3')
[SQL: SELECT id, btn, campaign
                            FROM rt_winback_existing_df rt, orders_order oo
                            WHERE oo.id == rt.order_id
                            AND date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"]
(Background on this error at: https://sqlalche.me/e/14/f405)
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:58:55
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:58:55] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.157:5000 (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM rt_winback_existing_df rt, orders_order oo
                            WHERE oo.id == rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:ROLLBACK
ERROR:routes.rt_winback:(pymysql.err.ProgrammingError) (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== rt.order_id' at line 3")
[SQL: SELECT id, btn, campaign
                            FROM rt_winback_existing_df rt, orders_order oo
                            WHERE oo.id == rt.order_id]
(Background on this error at: https://sqlalche.me/e/14/f405)
ERROR:routes.rt_winback:Exception thrown:(pymysql.err.ProgrammingError) (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== rt.order_id' at line 3")
[SQL: SELECT id, btn, campaign
                            FROM rt_winback_existing_df rt, orders_order oo
                            WHERE oo.id == rt.order_id]
(Background on this error at: https://sqlalche.me/e/14/f405)
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 21:59:40
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 21:59:40] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM rt_winback_existing_cb rt, orders_order oo
                            WHERE oo.id == rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:ROLLBACK
ERROR:routes.rt_winback:(pymysql.err.ProgrammingError) (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== rt.order_id' at line 3")
[SQL: SELECT id, btn, campaign
                            FROM rt_winback_existing_cb rt, orders_order oo
                            WHERE oo.id == rt.order_id]
(Background on this error at: https://sqlalche.me/e/14/f405)
ERROR:routes.rt_winback:Exception thrown:(pymysql.err.ProgrammingError) (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== rt.order_id' at line 3")
[SQL: SELECT id, btn, campaign
                            FROM rt_winback_existing_cb rt, orders_order oo
                            WHERE oo.id == rt.order_id]
(Background on this error at: https://sqlalche.me/e/14/f405)
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:00:37
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:00:37] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM rt_winback_existing_cb rt, orders_order oo
                            WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:01:41
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:01:41] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM orders_order 
                            WHERE date_installed >= "2020-01-01" 
                            AND date_installed <= "2022-04-09"
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                            FROM rt_winback_existing_cb rt, orders_order oo
                            WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:02:30
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:02:30] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.157:5000 (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= "2020-01-01" 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:07:00
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:07:00] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= "2020-01-01" 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:07:08
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:07:08] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= "2020-01-01" 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
ERROR:routes.helpers.db_manager:name 'results_new' is not defined
ERROR:routes.helpers.db_manager:Exception thrown:name 'results_new' is not defined
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:07:20
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:07:20] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= "2020-01-01" 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:07:27
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:07:27] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= "2020-01-01" 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:07:33
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:07:33] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= "2020-01-01" 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:07:35
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:07:35] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
INFO:werkzeug: * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.157:5000 (Press CTRL+C to quit)
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
ERROR:routes.rt_winback:get_all_orders() missing 1 required positional argument: 'self'
ERROR:routes.rt_winback:Exception thrown:get_all_orders() missing 1 required positional argument: 'self'
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:13:31] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
ERROR:routes.rt_winback:get_all_orders() missing 1 required positional argument: 'self'
ERROR:routes.rt_winback:Exception thrown:get_all_orders() missing 1 required positional argument: 'self'
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:13:54] "GET /launchrtwinback/ HTTP/1.1" 200 -
ERROR:routes.rt_winback:get_all_orders() missing 1 required positional argument: 'self'
ERROR:routes.rt_winback:Exception thrown:get_all_orders() missing 1 required positional argument: 'self'
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:13:56] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:13:56] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/rt_winback.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= '2020-01-01' 
                                AND date_installed <= '2022-04-09'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:15:13
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:15:13] "GET /launchrtwinback/ HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:werkzeug: * Detected change in '/Users/stutibiyani/Datastack/RTWinback/routes/helpers/db_manager.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 115-048-572
INFO:sqlalchemy.engine.Engine:SELECT *
                                FROM rt_winback_config
                                ORDER BY next_run_date LIMIT 1
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM orders_order 
                                WHERE date_installed >= '2022-04-09' 
                                AND date_installed <= '2022-04-14'
INFO:sqlalchemy.engine.Engine:[raw sql] {}
INFO:sqlalchemy.engine.Engine:SELECT id, btn, campaign
                                FROM rt_winback_existing_cb rt, orders_order oo
                                WHERE oo.id = rt.order_id
INFO:sqlalchemy.engine.Engine:[raw sql] {}
WARNING:routes.rt_winback:VERSION:2022-04-09 ---------------Starting new session--------------- on: 2022-04-09 22:15:29
WARNING:routes.rt_winback:---------------Ending  session--Time taken: 0:00:00
WARNING:routes.rt_winback:--------Time taken----: 0:00:00
INFO:werkzeug:127.0.0.1 - - [09/Apr/2022 22:15:29] "GET /launchrtwinback/ HTTP/1.1" 200 -
